helpers/mathjax.js

```diff
@@ -1,11 +1,16 @@
 let mathJaxLoaded;
 
 export async function htmlBlockMathRenderer(elem) {
-	const isLatexSupported = window.D2L && window.D2L.LP && window.D2L.LP.Web.UI.Flags.Flag('us125413-mathjax-render-latex', true);
+	const context = JSON.parse(document.documentElement.getAttribute('data-mathjax-context')) || {};
```

Comment:
This provider hasn't been created yet, so an LMS PR will be coming to handle that shortly. This also introduces an implicit dependency on FRAs - specifically, FRAs who want this functionality will need to update ifrau to at least 0.36.0 or they won't be able to take advantage of the `data-` attribute syncing.